artifacts/src/main/resources/issuance/credential-object-schema.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "title": "CredentialObjectSchema",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/definitions/CredentialObject"
+    }
+  ],
+  "$id": "https://w3id.org/dspace-dcp/v0.8/issuance/credential-object-schema.json",
+  "definitions": {
+    "CredentialObject": {
+      "type": "object",
+      "properties": {
+        "credentialType": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "offerReason": {
+          "type": "string"
+        },
+        "bindingMethods": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "cryptographicSuites": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "issuancePolicy": {
+          "type": "object"
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "@type": {
+              "type": "string",
+              "const": "CredentialObject"
+            }
+          },
+          "required": [
+            "@type"
+          ]
+        },
+        {
+          "properties": {
+            "type": {
+              "type": "string",
+              "const": "CredentialObject"
+            }
+          },
+          "required": [
+            "type"
+          ]
+        }
+      ],
+      "required": [
+        "credentialType",
```

Comment:
It's a spec bug. Opened an issue https://github.com/eclipse-dataspace-dcp/decentralized-claims-protocol/issues/90

Reply:
I think the Credential Object should not include the `@context` since it's not used as standalone object but rather than  embedded in `IssuerMetadata` and `CredentialOfferMessage`

Reply:
You're right, I'll change the scope of the issue.